tests/qpu/test_leaphybrid_common.py

```diff
@@ -97,8 +97,11 @@ class TestSamplesetInterface(unittest.TestCase):
 
     def test_wait_id_availability(self):
         # verify https://github.com/dwavesystems/dwave-system/issues/540 is fixed
```

Comment:
It is fixed, so do we still need this line?

Reply:
That comment describes the purpose to this test -- to verify `SampleSet` interface is consistent. So, yes.